README.md

```diff
@@ -21,3 +21,9 @@ Supported Mattermost Server Versions: 5.3+
 `/matterpoll "Is Matterpoll great?"` creates a poll with the answer options "Yes" and "No". You can also leave out the double quotes and just type `/matterpoll Is Matterpoll great?`.
 
 If you want to define all answer options by yourself, type `/matterpoll "Is Matterpoll great?" "Of course" "In any case" "Definitely"`- Note that the double quotes are required in this case.
+
+## Troubleshooting
+
+#### Pressing the poll buttons does nothing and creates a 400 error in the Mattermost log
+
+Make sure to set your [Site URL](https://docs.mattermost.com/administration/config-settings.html?highlight=site%20url#site-url) to use https.
```

Comment:
Trouble shooting section is good to us and users.

But always letting user set https to SiteURL looks misleading.  If running mattermost on http and setting `https://~` to SiteURL, I get the following 400 error message.

> Action integration error	{"path": "/api/v4/posts/ymfs6gpqz7f658ygyahh4c3ktw/actions/w7ny9dbsh7rrj8m1q1odyc6txa", "request_id": "wwhgdgjozffejcdixmbr8bqmie", "ip_addr": "::1", "user_id": "eqwpixku6tygjg1e9r9pkz1fye", "method": "POST", "err_where": "DoPostAction", "http_code": 400, "err_details": "err=Post https://localhost:8065/plugins/com.github.matterpoll.matterpoll/api/v1/polls/483m3uruhbytixfbntuq74x4zy/vote/0: http: server gave HTTP response to HTTPS client"}

So I suggest the following. 

```
Make sure to set your [Site URL](https://docs.mattermost.com/administration/config-settings.html?highlight=site%20url#site-url).
For example, this error happens in case you set SiteURL starting with `http://~`, in spite of running Mattermost server through https.
```

How does it sounds?

Reply:
Thanks for the feedback. I updated the text.